src/controllers/check.your.answers.controller.ts

```diff
@@ -56,7 +56,7 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
 
 export const post = async (req: Request, res: Response, next: NextFunction) => {
   try {
-    logger.debugRequest(req, `POST ${config.CHECK_YOUR_ANSWERS_PAGE}`);
+    logger.debugRequest(req, `POST ${config.CHECK_YOUR_ANSWERS_WITH_PARAMS_URL}`);
```

Comment:
the controller is used by the non-params url as well so it's probably best to leave it as config.CHECK_YOUR_ANSWERS_PAGE

Reply:
I have edited the controller back to config.CHECK_YOUR_ANSWERS_PAGE